src/lib.rs

```diff
@@ -127,6 +130,14 @@ impl EnvironmentType {
             EnvironmentType::Staging => 25,
         }
     }
+
+    pub fn get_default_private_node_count(&self) -> u16 {
+        match self {
+            EnvironmentType::Development => 1,
+            EnvironmentType::Production => 1,
```

Comment:
The `production.tfvars` for private node vm count is 0, but `variables.tf` is defaulted to 1.  I think .tfvars would take precedence over the default vm count of 1 for Production if passed in with `-var-file` etc option?

Any concern or issue here with Production reflecting 1 as oppose to 0 on the above line?

Reply:
Thanks for this, I'll update the tfvars file as mentioned in the above comment.

Reply:
This is the node count btw. The value in tfvars is for the VM count.

Reply:
Ahhaa I see. Thanks, then it should be equal to "generic node count"